CONTRIBUTING.md

```diff
@@ -140,43 +149,45 @@ CSS
 }
 ```
 
-- Filter and Filter+ work by inverting the whole web page and reverting necessary parts (images, videos, etc.), listed in the `INVERT` section.
-- If an inverted element contains images or other content that becomes wrongly displayed, `NO INVERT` rule can be used.
+- Filter and Filter+ modes work by inverting the whole web page and reverting necessary parts (images, videos, etc.) listed in the `INVERT` section.
+- If an inverted element contains images or other content that become wrongly displayed, the `NO INVERT` rule can be used.
 - `REMOVE BG` removes the background image from an element and forces a black background.
 
-## Add new features or fix bugs!
+## Add New Features or Fix Bugs!
+
+If you would like to **add a new feature** to Dark Reader or **fix a bug**, please **submit an issue** in GitHub (if there is no existing one), **discuss** it with active contributors, and wait for **approval**.
 
-If you would like to **add a new feature** to Dark Reader or **fix a bug**, **submit an issue** in GitHub (if there is no existing one), **discuss** it with active contributors, and wait for **approval**.
+#### To build and debug the extension:  
+1. Install the [Node.js](https://nodejs.org/) LTS.  
+1. Install development dependencies by running `npm install` in the project's root folder.  
+1. Execute `npm run debug`.
 
-To build and debug the extension **install the [Node.js](https://nodejs.org/)** LTS.  
-Install development dependencies by running `npm install` in the project root folder.  
-Then execute `npm run debug`.
+#### To install your test extension in Chrome and Edge
+1. Open the `chrome://extensions` page.
+1. Disable the official Dark Reader version.
+1. Enable the **Developer mode**.
+1. Click **Load unpacked extension** button.
+1. Select the test extension you built.
+1. Navigate to the project's `debug/` folder.
 
-#### Chrome and Edge
-- Open the `chrome://extensions` page.
-- Disable the official Dark Reader version.
-- Enable the **Developer mode**.
-- Click **Load unpacked extension** button
-- Navigate to the project's `debug/` folder.
+#### To install your test extension in Firefox
+1. Open the `about:addons` page.
```

Comment:
Repeating `1.`

Also this step plus the disable official one is unnecessary - Firefox handles this fine.